### PR TITLE
changed to golang native error wrapping

### PR DESCRIPTION
**Pull request by Debanitrkl**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:
Using the native error wrapping feature to omit the direct github.com/pkg/errors dependency.
#### Which issue(s) this PR fixes:

Related to #2549 
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
CI is still not happy :)

File: cmd/krel/cmd/announce_build.go [hunk 1]
```diff
@@ -238,28 +237,26 @@ func (opts *buildAnnounceOptions) saveAnnouncement(announcementSubject string, a
...
-		return errors.Wrap(err, "saving announcement-subject.txt")
-	}
+		return fmt.Errorf("saving announcement-subject.txt: %w", err)
 
 	logrus.Info("Kubernetes Announcement created.")
 	return nil
 }
 
-func getGoVersion() (string, error) {
+func getGoVersion (string, error) {
```
Comment (palnabarun) [inline cmd/krel/cmd/announce_build.go:253]: Removing the `()` is a typo.

File: cmd/krel/cmd/announce_build.go [hunk 2]
```diff
@@ -238,15 +237,14 @@ func (opts *buildAnnounceOptions) saveAnnouncement(announcementSubject string, a
...
+		return fmt.Errorf("saving announcement.html: %w", err)
 	}
 
 	absOutputPath = filepath.Join(opts.workDir, "announcement-subject.txt")
 	logrus.Infof("Writing announcement subject to %s", absOutputPath)
 	err = os.WriteFile(absOutputPath, []byte(announcementSubject), os.FileMode(0o644))
 	if err != nil {
-		return errors.Wrap(err, "saving announcement-subject.txt")
-	}
+		return fmt.Errorf("saving announcement-subject.txt: %w", err)
```
Comment (ArkaSaha30) [inline cmd/krel/cmd/announce_build.go]: Missing parenthesis for if-condition block

**Review by ArkaSaha30 — APPROVED** [review]

**Pull request merged**